blog/2024-06-01-barcamp-registration.md

```diff
@@ -0,0 +1,19 @@
+---
+author: Antonin Delpeuch
+title: "Register to participate to our BarCamp remotely"
+---
+
+Our first ever BarCamp is coming up in two weeks, on June 17th-19th, in Berlin and online.
+[Register now](https://barcamps.eu/openrefine-2024-barcamp/) to participate, whether you want to be onsite or remotely.
+This will give you access to the video conference room, let you propose and vote on sessions.
+
+The event will start Monday afternoon:
+* 14:00 CEST (12:00 UTC): Introductions round and opening of the event.
+* 14:30 CEST (12:30 UTC): Collaborative review and scheduling of [the proposed sessions](https://barcamps.eu/openrefine-2024-barcamp/sessions).
+* 16:00 CEST (14:00 UTC): Break
+* 16:30 CEST (14:30 UTC): Allocation of volunteering shifts (session facilitation, note-taking, Zoom caretaking)
```

Comment:
Random schedule made up when writing the blog post, we can adjust as you see fit.
@ayushrai206 I think it's easier to do the ice-breaker separately (later in the evening, separately for online and in-person attendees). See https://github.com/OpenRefine/BarCamp2024/issues/13

Reply:
Ah yes if it's indeed something rather short that can fit in an introduction round, then that makes sense (I thought you had something longer in mind)